app/src/main/java/org/oppia/app/player/state/StatePlayerRecyclerViewAssembler.kt

```diff
@@ -168,25 +169,58 @@ class StatePlayerRecyclerViewAssembler private constructor(
    * configuration of this assembler, as well as the GCS entity ID that should be associated with
    * rich-text rendering for this state.
    */
-  fun compute(ephemeralState: EphemeralState, gcsEntityId: String): List<StateItemViewModel> {
+  fun compute(
+    ephemeralState: EphemeralState,
+    gcsEntityId: String,
+    isSplitView: Boolean
+  ): Pair<List<StateItemViewModel>, List<StateItemViewModel>> {
     val hasPreviousState = ephemeralState.hasPreviousState
 
     previousAnswerViewModels.clear()
-    val pendingItemList = mutableListOf<StateItemViewModel>()
+    val leftPendingItemList = mutableListOf<StateItemViewModel>()
+    val rightPendingItemList = mutableListOf<StateItemViewModel>()
     if (playerFeatureSet.contentSupport) {
-      addContentItem(pendingItemList, ephemeralState, gcsEntityId)
+      addContentItem(leftPendingItemList, ephemeralState, gcsEntityId)
```

Comment:
Should we be using directional terminology (left/right)? How should this split column work for RTL layouts?

Perhaps instead, we have primary & secondary item lists, or conversation item list & extra interaction item list. What do you think?

Reply:
I'd go with "conversation item list & extra interaction item list"  
This is a screenshot from an RTL emulator
![screenshot-2020-07-24_10 32 46 199](https://user-images.githubusercontent.com/8533363/88374074-636d0180-cd99-11ea-8150-e250267c2446.png)
And using left/right for naming the lists isn't accurate, so I'll change them to " conversation item list & extra interaction item list"

Reply:
Done! used "conversation item list & extra interaction item list" teminology.

Reply:
Resolving this on behalf of Ben.